pyomo/repn/tests/cpxlp/test_lpv2.py

```diff
@@ -20,6 +20,72 @@
 from pyomo.repn.plugins.lp_writer import LPWriter
 
 
+def create_sos_model():
```

Comment:
This is a weak test case.  For verifying row order, a better case would have a Scalar and an Indexed SOSConstraint, (and the same for Constraint).  The we can test things like setting the order by passing the whole component, and by passing component datas separately. 

The 3 scalar constraints are redundant and unnecessary.